dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2683,7 +2683,7 @@ def test_split_row_groups_filter(tmpdir, engine):
         filters=filters,
     )
 
-    assert search_val in ddf3["i32"]
+    assert (ddf3["i32"] == search_val).any().compute()
```

Comment:
@rjzamora can you confirm that this is the desired check?

Reply:
This seems correct to me